local-volume/provisioner/pkg/common/common.go

```diff
@@ -169,26 +176,58 @@ func GetDefaultVolumeConfig() map[string]MountConfig {
 // VolumeConfigToConfigMapData converts volume config to configmap data.
 func VolumeConfigToConfigMapData(config map[string]MountConfig) (map[string]string, error) {
```

Comment:
The argument should be type ProvisionerConfiguration

Reply:
This function is also called in boostrapper, there we do not mount configmap, so the access to data is different. I am afraid that would require refactor bootstrapper.

Reply:
Refactoring bootstrapper is fine and will be required eventually if we add more configuration parameters.